index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+   <link href="https://fonts.googleapis.com/css2?family=Cuprum:wght@500&display=swap" rel="stylesheet">
+   <link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=PT+Sans&display=swap" rel="stylesheet">
+	<link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Cuprum:ital@1&display=swap" rel="stylesheet">
+	<title>Texnomart</title>
+</head>
+<body>
+	<header>
+		<div class="header1">
+					<button class="texno">техномарт</button>
+					<label for="user name" class="search">Поиск:</label>
```

Comment:
Добавьте поля input 
`<input type="search">`
```suggestion
					<label for="search-field" class="search">Поиск:</label>
                                      <input type="search" name="search" id="search-field">```